### PR TITLE
Merged work on the embed share state and overall refactoring / cleanup of embed css

### PR DESCRIPTION
**Pull request by coryshaw**
- Share State Styles
- Refactored embed to be more dry
- code cleanup of styles on controls.css

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/f14bf4db83ae1ac/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/f14bf4db83ae1ac/output.txt

Total script time: 1.09 mins

Server is running on http://pr844.butyr.org/templates/basic/
Run the Test Suite: http://pr844.butyr.org/test/

File: css/embed.less [hunk 1]
```diff
@@ -181,21 +181,41 @@ body {
...
-*/
+ SHARED STYLES BETWEEN POSTROLL AND SHARE STATES */
+
+
+#post-roll, #share {
 
-#post-roll {
   font-family: "Meta", "Open Sans", "Helvetica Neue", sans-serif;
-}
+  text-shadow: 1px 1px 0 rgba(255,255,255,0.4);
```
Comment (k88hudson) [inline css/embed.less]: Our style is 
`rgba( 255, 255, 255, 0.4 );`

File: css/embed.less [hunk 2]
```diff
@@ -181,21 +181,41 @@ body {
...
-  .post-roll-inner {
+  a {
+    color: #676867;
+  }
+
+  .post-roll-inner, .share-inner {
     position: relative;
     padding: 10%;
+    font-family: "Meta", "Open Sans", "Helvetica Neue", sans-serif;
+    text-shadow: 1px 1px 0px rgba(255,255,255,0.4);
```
Comment (k88hudson) [inline css/embed.less]: same thing here

File: css/embed.less [hunk 3]
```diff
@@ -304,85 +320,82 @@ body {
...
+  }
+  
+  .share-options fieldset {
+    border: none;
+    padding: 5px 0;
+  }
+  
+  .share-options label {
+    display: block;
+    padding-bottom: 0px;
```
Comment (k88hudson) [inline css/embed.less]: No `px` on 0 values

File: css/embed.less [hunk 4]
```diff
@@ -394,42 +407,14 @@ body {
...
-      line-height: 1em;
-    }
-
-    .mozpopcorn {
-      color: #D53B2A;
-      font-weight: bold;
-      font-size: 16px;
-      margin: 0 0 5px 0;
+      height:255px;
     }
```
Comment (k88hudson) [inline css/embed.less:518]: space after `:`

File: css/embed.less [hunk 5]
```diff
@@ -437,100 +422,159 @@ body {
...
-      font-size: 18px;
+
+    .embed-author {
+      font-size: 14px;
+    }
+
+    .embed-logo {
+      width: 20%;
+      margin-left: 20px;
+      margin-top: 0px;
```
Comment (k88hudson) [inline css/embed.less]: no `px` on 0 values

File: css/embed.less [hunk 6]
```diff
@@ -437,100 +422,159 @@ body {
...
+      height: 175px;
     }
+
     .embed-project {
-      padding-left: @embedLogoWidth + 30px;
+      margin-top: 0px;
```
Comment (k88hudson) [inline css/embed.less]: same thing here

File: css/embed.less (hunk 1 above)
Comment (coryshaw) [inline css/embed.less]: You're referring to the spacing? Is that just a Moz standard or are there other benefits to that spacing?

File: css/embed.less (hunk 3 above)
Comment (coryshaw) [inline css/embed.less]: Bad habit I need to break!

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e58b6f9e4e63b6e/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e58b6f9e4e63b6e/output.txt

Total script time: 0.74 mins

Server is running on http://pr844.butyr.org/templates/basic/
Run the Test Suite: http://pr844.butyr.org/test/

**Pull request closed without merging**